subprojects/platform-play/src/integTest/groovy/org/gradle/play/integtest/samples/UserGuidePlaySamplesIntegrationTest.groovy

```diff
@@ -118,6 +119,23 @@ class UserGuidePlaySamplesIntegrationTest extends AbstractIntegrationSpec {
             "router/RoutesPrefix.scala")
     }
 
+    @Requires(TestPrecondition.JDK8_OR_LATER)
```

Comment:
I prefer to duplicate the test here instead of using `where:` because I couldn't figure it out how to do that without changing `org.gradle.integtests.fixtures.AbstractIntegrationSpec.temporaryFolder` to be a `@ClassRule`, which would impact a lot of other tests.